dpdk/src/mem.rs

```diff
@@ -22,8 +22,10 @@ use errno::Errno;
 use tracing::{error, info, warn};
 
 // unfortunately, we need the standard library to swap allocators
+use dpdk_sys::{rte_pktmbuf_adj, rte_pktmbuf_prepend};
 use std::alloc::System;
```

Comment:
Nit: Keep comment above the relevant `use`?

Reply:
good catch.